api/orchestration_movers.go

```diff
@@ -452,3 +452,34 @@ func (o *datasyncOrchestrator) taskDetailsFromName(ctx context.Context, group, n
 
 	return nil, nil, apierror.New(apierror.ErrNotFound, "datasync mover not found", nil)
 }
+
+// taskRunsFromTaskArn finds a datasync task runs  based on its group/name and returns information about it
+func (o *datasyncOrchestrator) taskRunsFromTaskArn(ctx context.Context, taskArn string) ([]string, error) {
+	if taskArn == "" {
+		return nil, apierror.New(apierror.ErrBadRequest, "invalid input", nil)
+	}
+
+	// get a list of all datasync resources in the group
+	out, err := o.datasyncClient.ListDatasyncTasksexecutions(ctx, taskArn)
+	if err != nil {
+		return nil, err
+	}
+
+	//VISIT  : When 0 should return error or empty array
+	if len(out) == 0 {
```

Comment:
We can remove this check since we don't want to error, just return an empty slice

Reply:
Removed.